src/wp-includes/script-loader.php

```diff
@@ -212,6 +212,40 @@ function wp_default_packages_inline_scripts( $scripts ) {
 		),
 		'after'
 	);
+
+	$scripts->add_inline_script(
```

Comment:
Do we need this segment? ClassicPress v2 does not include the [WordPress data](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data/) package.

Reply:
I've just tested this, and you're right, so have removed that code from the PR.